mission_control/navigator_launch/nodes/network_listener.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+from sub8_alarm import single_alarm
+from twisted.internet import reactor
+import time
+
+
+class NetworkCheck(object):
```

Comment:
NetworkCheck would check for the network messages for the sub. There were two messages for the sub: `auto` and `kill`. A system like that may be set up for the boat in the future.